Source/Core/Core/HW/GCPadEmu.cpp

```diff
@@ -70,6 +70,8 @@ GCPad::GCPad(const unsigned int index) : m_index(index)
   // options
   groups.emplace_back(m_options = new ControlGroup(_trans("Options")));
   m_options->boolean_settings.emplace_back(
+      m_always_connected = new ControlGroup::BooleanSetting(_trans("Always Connected"), false));
```

Comment:
Please add an i18n comment that says that "connected" refers to a controller. This might help translators pick the right declension of the word "connected".

An i18n comment starts with `// i18n: `, and as far as I know, it always needs to be on the line before the string. So in this case, it would need to be on the line above "Always Connected", not on the line above m_options->...